test_sitemap_utils.py

```
@@ -109,25 +108,25 @@ def side_effect(url):
-                """
-            elif url == "https://example.com/custom_sitemap.xml":
-                return """<?xml version="1.0" encoding="UTF-8"?>
+                """,
+            "https://example.com/custom_sitemap.xml": """<?xml version="1.0" encoding="UTF-8"?>
                 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
                     <url>
                         <loc>https://example.com/home</loc>
                         <priority>1.0</priority>
                     </url>
                 </urlset>
                 """
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)